atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1619,7 +1484,7 @@ public Void apply(Client client) throws NotFoundException, InvalidRequestExcepti
                 }
             });
         } catch (Exception e) {
-            log.error("Exception encountered while setting gc_grace_seconds:{} and tombstone_threshold:{} for {}.{}",
+            log.error("Exception encountered while setting gc_grace_seconds:{} and tombstone_threshold:{} for {}. {}",
```

Comment:
I imagined it would look pretty weird as "atlas.gltn.ctx_foo_obj" but I guess that works.